### ISSUE TITLE
Sufficient Velocity not working

### ISSUE THREAD

**Issue by Twilight666** [JimmXinu/FanFicFare#399]
I have just tried the CLI version for these

> https://forums.sufficientvelocity.com/threads/fallout-bug-bay-worm-fallout-crossover.38200/
> https://forums.sufficientvelocity.com/threads/boom-worm-minor-au.54623/

and I get the following
`URL(https://forums.sufficientvelocity.com/threads/fallout-bug-bay-worm-fallout-crossover.38200/) Failed: Exception (Unknown Site(https://forums.sufficientvelocity.com/threads/fallout-bug-bay-worm-fallout-crossover.38200/).  Supported sites:..............`
So for some reason SV is not recognised

**Comment by JimmXinu** [comment]
What version of FFF are you using?  What version of Python?  On what OS?

Did SV appear in the supported list you left out?

Have you run `--debug` to get more information?  (Bonus: Debug output includes FFF, Python and OS versions.)

Kudos for including story URLs, but I need more info.

**Comment by Twilight666** [comment]
I am using FFF 3.7 on Python3 on Windows 10
Here is the individual error:
```
Traceback (most recent call last):
  File "c:\Python3.7\Scripts\fanficfare-script.py", line 11, in <module>
    load_entry_point('FanFicFare==3.7.0', 'console_scripts', 'fanficfare')()
  File "c:\python3.7\lib\site-packages\fanficfare\cli.py", line 290, in main
    passed_personalini)
  File "c:\python3.7\lib\site-packages\fanficfare\cli.py", line 338, in do_download
    output_filename)
  File "c:\python3.7\lib\site-packages\fanficfare\cli.py", line 524, in get_configuration
    raise e
  File "c:\python3.7\lib\site-packages\fanficfare\cli.py", line 518, in get_configuration
    configuration = Configuration(adapters.getConfigSectionsFor(url), options.format)
  File "c:\python3.7\lib\site-packages\fanficfare\adapters\__init__.py", line 278, in getConfigSectionsFor
    raise exceptions.UnknownSite( url, [cls.getSiteDomain() for cls in __class_list] )
```
And the full list of sites is:
`fanficfare.exceptions.UnknownSite: Unknown Site(https://forums.sufficientvelocity.com/threads/fallout-bug-bay-worm-fallout-crossover.38200/).  Supported sites: (archive.shriftweb.org, archive.skyehawke.com, archiveofourown.org, ashwinder.sycophanthex.com, bloodshedverse.com, bloodties-fans.com, buffygiles.velocitygrass.com, chaos.sycophanthex.com, chosentwofanfic.com, csi-forensics.com, dark-solace.org, efiction.esteliel.de, efpfanfic.net, erosnsappho.sycophanthex.com, fanfic.castletv.net, fanfic.hu, fanfic.potterheadsanonymous.com, fanfiction-junkies.de, fannation.shades-of-moonlight.com, fictionhunt.com, fictionmania.tv, fictionpad.com, ficwad.com, finestories.com, forum.questionablequesting.com, forums.spacebattles.com, gluttonyfiction.com, gravitytales.com, harrypotterfanfiction.com, hlfiction.net, hpfanficarchive.com, imagine.e-fic.com, inkbunny.net, it-could-happen.net, ksarchive.com, lcfanfic.com, literotica.com, lotrfanfiction.com, lumos.sycophanthex.com, mcstories.com, merlinfic.dtwins.co.uk, mtt.just-once.net, ncisfiction.com, nfacommunity.com, nha.magical-worlds.us, ninelivesarchive.com, novelonlinefull.com, noveltrove.com, occlumency.sycophanthex.com, ponyfictionarchive.net, pretendercentre.com, quotev.com, samandjack.net, sebklaine.net, sheppardweir.com, sinful-dreams.com, sinful-dreams.com, sinful-dreams.com, spikeluver.com, starslibrary.net, storiesonline.net, sword.borderline-angel.com, tasteofpoison.inkubation.net, test1.com, tgstorytime.com, thehookupzone.net, themaplebookshelf.com, themasque.net, trekfanfiction.net, trekiverse.org, voracity2.e-fic.com, www.13hours.org, www.adastrafanfic.com, www.adult-fanfiction.org, www.alternatehistory.com, www.andromeda-web.com, www.aneroticstory.com, www.asexstories.com, www.asianfanfics.com, www.bdsmlibrary.com, www.deandamage.com, www.deepinmysoul.net, www.destinysgateway.com, www.dokuga.com, www.dracoandginny.com, www.fanficauthors.net, www.fanfiction.net, www.fanfiktion.de, www.ficbook.net, www.fictionalley.org, www.fictionpress.com, www.fimfiction.net, www.fireflyfans.net, www.giantessworld.net, www.hentai-foundry.com, www.ik-eternal.net, www.libraryofmoria.com, www.looselugs.com, www.lotrgfic.com, www.lushstories.com, www.masseffect2.in, www.mediaminer.org, www.midnightwhispers.net, www.mugglenetfanfiction.com, www.naiceanilme.net, www.narutofic.org, www.novelall.com, www.phoenixsong.net, www.potionsandsnitches.org, www.potterfics.com, www.qaf-fic.com, www.royalroad.com, www.scarvesandcoffee.net, www.siye.co.uk, www.squidge.org, www.starskyhutcharchive.net, www.storiesofarda.com, www.sugarquill.net, www.thedelphicexpanse.com, www.thepetulantpoetess.com, www.thundercatsfans.org, www.tomparisdorm.com, www.tthfanfic.org, www.twilightarchives.com, www.twilighted.net, www.utopiastories.com, www.valentchamber.com, www.walkingtheplank.org, www.wattpad.com, www.webnovel.com, www.whofic.com, www.wolverineandrogue.com, www.wraithbait.com, www.wuxiaworld.co, www.wuxiaworld.com, xf2test.sufficientvelocity.com)`

**Comment by Twilight666** [comment]
Also here

> FFF: DEBUG: 2019-05-02 21:48:25,412: cli.py(184):     OS Version:Windows-10-10.0.17763-SP0
> FFF: DEBUG: 2019-05-02 21:48:25,412: cli.py(185): Python Version:3.7.2 (tags/v3.7.2:9a3ffc0492, Dec 23 2018, 22:20:52) [MSC v.1916 32 bit (Intel)]
> FFF: DEBUG: 2019-05-02 21:48:25,412: cli.py(186):    FFF Version:3.7.0
> Traceback (most recent call last):
>   File "c:\Python3.7\Scripts\fanficfare-script.py", line 11, in <module>
>     load_entry_point('FanFicFare==3.7.0', 'console_scripts', 'fanficfare')()
>   File "c:\python3.7\lib\site-packages\fanficfare\cli.py", line 290, in main
>     passed_personalini)
>   File "c:\python3.7\lib\site-packages\fanficfare\cli.py", line 338, in do_download
>     output_filename)
>   File "c:\python3.7\lib\site-packages\fanficfare\cli.py", line 524, in get_configuration
>     raise e
>   File "c:\python3.7\lib\site-packages\fanficfare\cli.py", line 518, in get_configuration
>     configuration = Configuration(adapters.getConfigSectionsFor(url), options.format)
>   File "c:\python3.7\lib\site-packages\fanficfare\adapters\__init__.py", line 278, in getConfigSectionsFor
>     raise exceptions.UnknownSite( url, [cls.getSiteDomain() for cls in __class_list] )
> fanficfare.exceptions.UnknownSite: Unknown Site(https://forums.sufficientvelocity.com/threads/fallout-bug-bay-worm-fallout-crossover.38200/).  Supported sites: (archive.shriftweb.org, archive.skyehawke.com, archiveofourown.org, ashwinder.sycophanthex.com, bloodshedverse.com, bloodties-fans.com, buffygiles.velocitygrass.com, chaos.sycophanthex.com, chosentwofanfic.com, csi-forensics.com, dark-solace.org, efiction.esteliel.de, efpfanfic.net, erosnsappho.sycophanthex.com, fanfic.castletv.net, fanfic.hu, fanfic.potterheadsanonymous.com, fanfiction-junkies.de, fannation.shades-of-moonlight.com, fictionhunt.com, fictionmania.tv, fictionpad.com, ficwad.com, finestories.com, forum.questionablequesting.com, forums.spacebattles.com, gluttonyfiction.com, gravitytales.com, harrypotterfanfiction.com, hlfiction.net, hpfanficarchive.com, imagine.e-fic.com, inkbunny.net, it-could-happen.net, ksarchive.com, lcfanfic.com, literotica.com, lotrfanfiction.com, lumos.sycophanthex.com, mcstories.com, merlinfic.dtwins.co.uk, mtt.just-once.net, ncisfiction.com, nfacommunity.com, nha.magical-worlds.us, ninelivesarchive.com, novelonlinefull.com, noveltrove.com, occlumency.sycophanthex.com, ponyfictionarchive.net, pretendercentre.com, quotev.com, samandjack.net, sebklaine.net, sheppardweir.com, sinful-dreams.com, sinful-dreams.com, sinful-dreams.com, spikeluver.com, starslibrary.net, storiesonline.net, sword.borderline-angel.com, tasteofpoison.inkubation.net, test1.com, tgstorytime.com, thehookupzone.net, themaplebookshelf.com, themasque.net, trekfanfiction.net, trekiverse.org, voracity2.e-fic.com, www.13hours.org, www.adastrafanfic.com, www.adult-fanfiction.org, www.alternatehistory.com, www.andromeda-web.com, www.aneroticstory.com, www.asexstories.com, www.asianfanfics.com, www.bdsmlibrary.com, www.deandamage.com, www.deepinmysoul.net, www.destinysgateway.com, www.dokuga.com, www.dracoandginny.com, www.fanficauthors.net, www.fanfiction.net, www.fanfiktion.de, www.ficbook.net, www.fictionalley.org, www.fictionpress.com, www.fimfiction.net, www.fireflyfans.net, www.giantessworld.net, www.hentai-foundry.com, www.ik-eternal.net, www.libraryofmoria.com, www.looselugs.com, www.lotrgfic.com, www.lushstories.com, www.masseffect2.in, www.mediaminer.org, www.midnightwhispers.net, www.mugglenetfanfiction.com, www.naiceanilme.net, www.narutofic.org, www.novelall.com, www.phoenixsong.net, www.potionsandsnitches.org, www.potterfics.com, www.qaf-fic.com, www.royalroad.com, www.scarvesandcoffee.net, www.siye.co.uk, www.squidge.org, www.starskyhutcharchive.net, www.storiesofarda.com, www.sugarquill.net, www.thedelphicexpanse.com, www.thepetulantpoetess.com, www.thundercatsfans.org, www.tomparisdorm.com, www.tthfanfic.org, www.twilightarchives.com, www.twilighted.net, www.utopiastories.com, www.valentchamber.com, www.walkingtheplank.org, www.wattpad.com, www.webnovel.com, www.whofic.com, www.wolverineandrogue.com, www.wraithbait.com, www.wuxiaworld.co, www.wuxiaworld.com, xf2test.sufficientvelocity.com)

**Comment by JimmXinu** [comment]
Seeing that you have `FFF Version:3.7.0` and that `xf2test.sufficientvelocity.com` appears on the supported list tells me that I some how managed to upload a development version (xf2test is a future upgraded version of SV) in place of the regular update.

I'm not sure when I would have done that, but I'm the only one who could have. :-/

I've released v3.7.6 on CLI only.  It contains a few fixes from v3.7.6 and corrects the xf2test issue.  Do a `pip` upgrade and you should be fine again.

**Comment by Twilight666** [comment]
Ok I tried the newest test version and it worked
